### DISCUSSION TITLE
Setting custom capacity when new and max range won't work for multiple Teslas

### DISCUSSION THREAD

**Discussion by rhinoTV** [jheredianet/Teslamate-CustomGrafanaDashboards#53] [General]
We have three Teslas in our Tesla account, Model S P100D, Model X P100D, and a Model 3 RWD LR.  When I set the original battery capacity for the X, it uses the same value for all 3 cars. This is incorrect for the Model 3. Can each car have its own setting for these variables?

**Comment by jheredianet** [comment]
Hi, I think you mean that when using the dropdown list, the variables remain on what they have. That's a limitation we have since in TeslaMate database we don't have custom fields where to save those kinds of values. 

A work around for this is to save a bookmark (or favorite) in your browser. So, select your first car, enter the corresponding values then save a bookmark in your browser and label it as you wish, then do the same for the rest. 
Next time, instead of open your dashboard from TeslaMate, just open directly from your bookmark.

**Reply by rhinoTV** [reply]
Yes. That was exactly what I meant. Thank you for the workaround. Too bad there's not a more elegant way to do it. Also thank you for these custom dashboards!
